lizmap/modules/dataviz/classes/datavizPlot.class.php

```diff
@@ -136,53 +136,6 @@ private function parsePlotConfig($plotConfig)
                     $z_fields[] = $trace->z_field;
                 }
             }
-        } else {
-            // LEGACY CODE:  LIZMAP < 3.4
```

Comment:
We need to keep that to read CFG file made with < 3.4.
Not all CFG are updated. Otherwise, we need from our users to open **all** projects on a server, open Lizmap and save lizmap. They are plenty of projects on a server.

Reply:
I see. I'll handle that.

Reply:
To remove this kind of code, you need to provide a table with all Lizmap projects and their version.

For now, there is no way to know, in a long list of lizmap projects sitting on a server, each LWC target version.